### DISCUSSION TITLE
주문 내역

### DISCUSSION THREAD

**Discussion by ParkSeoJune** [daon1213/Goj_Gam#3] [Q&A] [answered]
MyPage 주문 내역에 주문한 식당 이름이 아닌 주문 데이터가 나오는 것이 궁금합니다.

**Comment by daon1213** [comment, marked as answer]
Firebase RTDB에 담긴 Order 데이터의 번호를 그대로 받아오기 때문에 그렇게 보이는데 추후 그 데이터를 받아와 String으로 변환하여
주문 식당의 이름이나 주문한 상품 이름이 나오게끔 변경하겠습니다.
